optuna/importance/_fanova.py

```diff
@@ -121,12 +117,3 @@ def _distribution_to_hyperparameter(name: str, distribution: BaseDistribution) -
             "The parameter distribution should be one of the {}".format(d, distribution_list)
         )
     return hp
-
-
-def _check_fanova_availability() -> None:
-    if not _available:
-        raise ImportError(
-            "fanova is not available. Please install automl/fanova to use this feature. "
-            "For further information, please refer to the installation guide of automl/fanova. "
-            "(The actual import error is as follows: " + str(_import_error) + ")."
-        )
```

Comment:
This evaluator will be removed by #1106, and this is an optional comment.
IMO, this information is also useful because `fanova` cannot be installed via PyPI. How about keeping it?

The current message is as follows:
```console
Traceback (most recent call last):
  File "/Users/yanase/pfn/code/optuna/optuna/importance/_fanova.py", line 26, in <module>
    from fanova import fANOVA
ModuleNotFoundError: No module named 'fanova'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/yanase/pfn/code/optuna/optuna/_experimental.py", line 86, in wrapped_init
    _original_init(self, *args, **kwargs)
  File "/Users/yanase/pfn/code/optuna/optuna/importance/_fanova.py", line 48, in __init__
    _imports.check()
  File "/Users/yanase/pfn/code/optuna/optuna/_imports.py", line 80, in check
    raise exc_type(self._message) from exc_value
ModuleNotFoundError: Failed to import an optional package. See causing exceptions for details.
```

Reply:
Do you mean not to use `try_import`? `fanova` is available on PyPI (although it requires swig).